### ISSUE TITLE
with latest commit there is an issue while constructing table name

### ISSUE THREAD

**Issue by manib-kumar** [go-xorm/xorm#881]
```
type MCC struct {
	ID          int64  `xorm:"pk 'id'"`
	Code        string `xorm:"'code'"`
	Description string `xorm:"'description'"`
}

func (mcc *MCC) TableName() string {
	return "mcc"
}
```

here I am getting table name as "m_c_c" instead of "mcc". It is working fine for older commit.
```
name: github.com/go-xorm/xorm
version: 11743c1a80897b0fdbf79d88ebcaa7dd5f9d3cf3
```

**Comment by lunny** [comment]
@manib-kumar please see #885, your issue cannot be reproduced.

**Comment by lunny** [comment]
should be closed by #885
